pkg/api/timestamp.go

```diff
@@ -45,6 +49,22 @@ func RequestFromRekor(ctx context.Context, req pkcs9.TimeStampReq) ([]byte, erro
 	return body, nil
 }
 
+func createRFC3161FromBytes(resp []byte) models.ProposedEntry {
```

Comment:
This feels a little out of place here (vs being implemented inside the actual type and being called here), but this would certainly work.

Reply:
addressed, now I share code between pflags and here